third_party/2and3/typing_extensions.pyi

```diff
@@ -59,3 +59,11 @@ if sys.version_info >= (3, 5):
 
 if sys.version_info >= (3, 6):
     from typing import AsyncGenerator as AsyncGenerator
+
+def get_type_hints(
+    obj: Callable[..., Any], globalns: Optional[Dict[str, Any]] = ..., localns: Optional[Dict[str, Any]] = ...,
+    include_extras: bool = ...
+) -> Dict[str, Any]: ...
+
+Annotated: _SpecialForm = ...
+_AnnotatedAlias: Any = ...
```

Comment:
This is a private thing and doesn't really need to be here. If you actually need it for something, then please add an `# undocumented` comment.

Reply:
You're right – let me explain why I added it, there should be a better way to achieve what I need: I have some code that needs to check if an object is an instance of `Annotated`. What I have right now:

```python
from typing_extensions import _AnnotatedAlias, Annotated

def fun(x):
    if isinstance(x, _AnnotatedAlias):
        # ...
    else:
        # ...

fun(int)
fun(Annotated[int, 'whatever'])
```

`get_origin` introduced in `typing` in Python 3.8 could help here but it's not modified to know about `Annotated` yet. I'm happy to remove `_AnnotatedAlias` from the stub and silence mypy error though if we don't want to pollute stubs with unwanted things.

Reply:
I think it is fine if you add `# undocumented` here, see https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#what-to-include